### PR TITLE
Close #3377 require "Access Draggableviews" permission to access carousel items draggable view.

### LINKED ISSUE #3377: Reorder Carousel functionality is public
## Problem/Motivation
The reorder carousel page is publicly accessible.

### PR DESCRIPTION
**Pull request by trackleft**
## Description
In this pull request I change the access control settings to be set per display.
I also add a new permissions requirement of "Access draggable views" which both content editor and content admin have associated with.

image
image
image

## Related issues
#3377 

## How to test
See https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-3378.probo.build/user?destination=/admin/az-carousel/reorder-carousel-items
See https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-3378.probo.build/user?destination=/admin/az-carousel/reorder-carousel-items
### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
   - [ ] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
   - [ ] Adding experimental module
   - [ ] Update experimental module
- **Minor release changes**
   - [ ] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Upgrade experimental module to stable
   - [ ] Enable existing module by default or database update
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- **Other or unknown**
   - [ ] Other or unknown

### Drupal core
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major or minor level update
- **Other or unknown**
   - [ ] Other or unknown

### Drupal contrib projects
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major level update
- **Other or unknown**
   - [ ] Other or unknown

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by tadean — DISMISSED** [review]

**Review by danahertzberg — DISMISSED** [review]

File: modules/custom/az_carousel/config/install/views.view.az_carousel.yml [hunk 1]
```diff
@@ -106,7 +106,7 @@ display:
       access:
         type: perm
         options:
-          perm: 'access content'
+          perm: 'access draggableviews'
```
Comment (trackleft) [inline modules/custom/az_carousel/config/install/views.view.az_carousel.yml:109]:
```suggestion
          perm: 'access content'
```

**Comment by trackleft** [comment]
See https://9d0532dc-9753-4996-8fef-e4697e9c6c4d.probo.build/admin/structure/views/view/az_carousel/edit/reorder_carousel_items

File: modules/custom/az_carousel/config/install/views.view.az_carousel.yml [hunk 2]
```diff
@@ -601,7 +610,6 @@ display:
         fields: false
         sorts: false
         pager_options: false
-        footer: false
```
Comment (joeparsons) [inline modules/custom/az_carousel/config/install/views.view.az_carousel.yml:604]: Any idea what this `footer` change is about?  Just want to make sure it's not undoing anything from #3363

Comment (trackleft) [reply modules/custom/az_carousel/config/install/views.view.az_carousel.yml:604]: Yeah, the footer was added here: https://github.com/az-digital/az_quickstart/commit/1b175ea19531325f7585d585556d2ce751867cf7#diff-59a762683ffc2801f3f36e5b61c632fef90614d7ed22155af20edaf893ed7f3bR609

Comment (trackleft) [reply modules/custom/az_carousel/config/install/views.view.az_carousel.yml:604]: https://9d0532dc-9753-4996-8fef-e4697e9c6c4d.probo.build/user/login?destination=/admin/az-carousel/reorder-carousel-items

**Review by tadean — APPROVED** [review]

**Review by joeparsons — APPROVED** [review]

**Pull request merged**